### PR TITLE
Spring-Zeebe: description update

### PR DESCRIPTION
**Pull request by aleksander-dytko**
Updated README.md to communicate about the upcoming changes and new supported SDK for Spring Zeebe

### CODE REVIEW

**Review by MaxTru — CHANGES_REQUESTED** [review]
See comment

**Update:** my comment got lost? Will re-do.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-# Spring Zeebe -> Camunda Spring SDK
+# Spring-Zeebe -> Camunda Spring Zeebe SDK
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.camunda/spring-zeebe/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.camunda/spring-zeebe)
 [![Project Stats](https://www.openhub.net/p/spring-zeebe/widgets/project_thin_badge.gif)](https://www.openhub.net/p/spring-zeebe)
 
-Spring Zeebe will slowly evolve towards Camunda Spring SDK.
+Spring-Zeebe (this repository) will evolve into Spring Zeebe SDK in the [Zeebe repository](https://github.com/camunda/zeebe) that will be officially supported.
+
+Later on, Camunda will expand Spring Zeebe SDK to deliver a Camunda Spring SDK providing a unified experience for interacting with all Camunda APIs in the automation cluster (Operate, Tasklist, Optimize and Identity) in Java Spring. We recommend migrating to the officially supported Spring Zeebe and Camunda Spring SDK.
```
Comment (MaxTru) [inline README.md:10]: I would:

* Put this into disclaimer format
* Only highlight the key info: this is not supported, the supported one is somewhere else
* The info that the Zeebe SDK evolves into CAmunda SDK, I would put in the docs or into the zeebe repo

For example:

```suggestion
> [!IMPORTANT]  
> This is the community maintained version of the Spring Zeebe SDK. For the officially Camunda supported version of this SDK, please refer to the [Zeebe repository](https://github.com/camunda/zeebe).
```

Comment (aleksander-dytko) [reply README.md:10]: Thanks @MaxTru, I like that!

Comment (akeller) [reply README.md]: We need to add more context here based on issues and discussions internally and externally. I will open a new PR with similar language to what @aleksander-dytko originally authored. 

Happy to follow up with you on this @MaxTru.

**Pull request merged**